example.rb

```diff
@@ -3,5 +3,8 @@ class Example
   def rails_conf
     puts "RailsConf 2015!"
   end
+  def hello_world
+    puts 'Hello World!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.